internal/wazeroir/compiler.go

```diff
@@ -731,20 +727,18 @@ operatorSwitch:
 		targetFrame := c.controlFrames.get(int(targetIndex))
 		targetFrame.ensureContinuation()
 		drop := c.getFrameDropRange(targetFrame, false)
-		target := targetFrame.asLabel()
-		c.result.LabelCallers[target.ID()]++
+		targetID := targetFrame.asLabelID()
+		c.result.LabelCallers[targetID]++
 
-		continuationLabel := Label{FrameID: c.nextID(), Kind: LabelKindHeader}
-		c.result.LabelCallers[continuationLabel.ID()]++
+		continuationLabel := NewLabelID(LabelKindHeader, c.nextID())
+		c.result.LabelCallers[continuationLabel]++
 		c.emit(
-			OperationBrIf{
-				Then: BranchTargetDrop{ToDrop: drop, Target: target},
-				Else: continuationLabel.asBranchTargetDrop(),
-			},
+			NewOperationBrIf(
+				/* then */ BranchTargetDrop{ToDrop: drop, Target: targetID},
+				/* else */ continuationLabel.asBranchTargetDrop(),
```

Comment:
no need to name params.
```suggestion
				BranchTargetDrop{ToDrop: drop, Target: targetID},
				continuationLabel.asBranchTargetDrop(),
```